README.md

```diff
@@ -334,7 +336,7 @@ We'll use this as an argument to `dropdown`:
 d <- dropdown Times (constDyn ops) def
 ```
 
-We are using `constDyn` again here to turn our `Map` of operations into a `Dynamic`. Using `def`, we provide the default `DropdownConfig`. The result, `d`, will be a `Dropdown`. We can retrieve the `Dynamic` selection of a `Dropdown` by using `_dropdown_value`.
+We are using `constDyn` here to turn our `Map` of operations into a `Dynamic`. Using `def`, we provide the default `DropdownConfig`. The result, `d`, will be a `Dropdown`. We can retrieve the `Dynamic` selection of a `Dropdown` by using `_dropdown_value`.
```

Comment:
constDyn is mentioned for the first time in this paragraph, so seeing again here was confusing.